ui/src/app/job-details/resources/resources.component.html

```diff
@@ -20,5 +20,10 @@
         {{ logFileData.get(logFile) }}
       </div>
     </div>
+    <div *ngIf="currentTabId == 'event-detail'" class="inline-text">
+      <div *ngFor="let eventDetail of eventDetails" class="inline-text event-detail">
```

Comment:
Is the "inline-text" class needed on this line? (you already have it on the containing div)

Reply:
yes, this gives a pre space on the whole paragraph.